src/Components/Server/src/Builder/ComponentEndpointRouteBuilderExtensions.cs

```diff
@@ -28,6 +29,28 @@ public static ComponentEndpointConventionBuilder MapBlazorHub(this IEndpointRout
             return new ComponentEndpointConventionBuilder(endpoints.MapHub<ComponentHub>(ComponentHub.DefaultPath));
         }
 
+        /// <summary>
+        /// Maps the SignalR <see cref="ComponentHub"/> to the path <see cref="ComponentHub.DefaultPath"/>.
```

Comment:
`ComponentHub` is internal no? We should avoid including it here since users wouldn't know what it is.

> Maps the Blazor `<see cref="Hub" />` to the default path.

Reply:
Same update elsewhere